src/main/java/org/mockito/session/MockitoSessionBuilder.java

```diff
@@ -55,6 +85,21 @@
     @Incubating
     MockitoSessionBuilder strictness(Strictness strictness);
 
+    /**
+     * Configures logger used by {@code MockitoSession} for emitting hints when finishing
```

Comment:
Can you add javadoc link to MockitoHint class? This will help getting a big picture what is the deal with hints.

Reply:
👍